deployment/src/main/java/io/quarkiverse/openapi/generator/deployment/template/OpenApiNamespaceResolver.java

```diff
@@ -0,0 +1,59 @@
+package io.quarkiverse.openapi.generator.deployment.template;
```

Comment:
as you know, This file is not longer needed, you can add the getDeprecatedModelAttr to  https://github.com/quarkiverse/quarkus-openapi-generator/blob/main/deployment/src/main/java/io/quarkiverse/openapi/generator/deployment/template/QuteTemplatingExtension.java

Reply:
Actually, I'll keep the naming  `OpenApiNamespaceResolver` since it has a clear definition of what it is.